### PR TITLE
feat(boxai-sidebar): Pass isStopResponseEnabled prop to BoxAiContentAnswers

### PR DESCRIPTION
**Pull request by DanilaRubleuski**
### Description 
Passed  isStopResponseEnabled prop to BoxAiContentAnswers, so the AI in sidebar has the same stop response experience as modal.

### Recording

https://github.com/user-attachments/assets/758341d4-bcbb-4d60-b15d-d46e9660c4df

### CODE REVIEW

**Review by marcoartaviaq — APPROVED** [review]
Do we still plan to embed the stop inside the Ask button? I know is out of scope for this PR, but wondering the future of it.

File: src/elements/content-sidebar/BoxAISidebar.tsx [hunk 1]
```diff
@@ -13,69 +13,74 @@ import { DOCUMENT_SUGGESTED_QUESTIONS, SPREADSHEET_FILE_EXTENSIONS } from '../co
...
-    contentName: string,
-    elementId: string,
-    recordAction: (params: RecordActionType) => void,
-    setCacheValue: (key: 'encodedSession' | 'questions', value: string | null | QuestionType[]) => void,
-    userInfo: { name: string, avatarUrl: string },
-};
+    cache: { encodedSession?: string | null; questions?: QuestionType[] };
+    contentName: string;
+    elementId: string;
+    isStopResponseEnabled: boolean;
```
Comment (karolinaru) [inline src/elements/content-sidebar/BoxAISidebar.tsx:19]: wondering shouldn't it be optional if we then set the default value to false?

Comment (karolinaru) [reply src/elements/content-sidebar/BoxAISidebar.tsx:19]: and is it possible to test it in storybook that stop button appears once question is sent?

**Review by marcoartaviaq — APPROVED** [review]

**Review by kkuliczkowski-box — APPROVED** [review]

**Comment by tjuanitas** [comment]
force merging this since a datetime unit test is failing because of the new year

**Pull request merged**